intelij-plugin/src/main/java/tech/pantheon/yanginator/plugin/formatter/YangFormatterUtils.java

```diff
@@ -0,0 +1,101 @@
+package tech.pantheon.yanginator.plugin.formatter;
+
+import com.intellij.formatting.Indent;
+import com.intellij.psi.tree.IElementType;
+import com.intellij.psi.tree.TokenSet;
+import tech.pantheon.yanginator.plugin.psi.YangTypes;
+
+public class YangFormatterUtils {
```

Comment:
This looks like a utility class. Make it final (so that it's clear it should not be extended) and add a private constructor (so that it's clear it should not be instantiated). Also it looks like it's used only in the tech.pantheon.yanginator.plugin.formatter package, so it does not need to be public.